core/lib/types/src/api/mod.rs

```diff
@@ -740,35 +739,16 @@ pub struct Proof {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct OptimisticTransactionResult {
+pub struct TransactionDetailedResult {
     pub transaction_hash: H256,
     pub storage_logs: Vec<ApiStorageLogQuery>,
-    pub events: Vec<ApiVmEvent>,
-}
-
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
-pub struct ApiVmEvent {
-    pub l1_batch_number: L1BatchNumber,
-    pub tx_batch_index: u32,
-    pub address: Address,
-    pub indexed_topics: Vec<H256>,
-    pub value: String,
+    pub events: Vec<Log>,
 }
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct ApiStorageLogQuery {
```

Comment:
let's rename to `ApiStorageLog` (`Query` comes from crypto code and I think it's out of place here)

Reply:
Renamed